docs/en/operation/User-CLI.md

```diff
@@ -87,13 +110,15 @@ $ ./bin/alluxio formatWorker
 ### bootstrapConf
 
 The `bootstrapConf` command generates the bootstrap configuration file
-`${ALLUXIO_HOME}/conf/alluxio-env.sh` with the specified `ALLUXIO_MASTER_HOSTNAME`,
+`${ALLUXIO_HOME}/conf/alluxio-site.properties` with the specified `ALLUXIO_MASTER_HOSTNAME`,
 if the configuration file does not exist.
 
+<!-- Configuration file is empty except for "alluxio.master.hostname"; Alluxio will use default settings
 In addition, worker memory size and the ramdisk folder will be set in the configuration file
 in accordance to the state of the machine:
 * type: Mac or Linux
 * total memory size
+-->
```

Comment:
I can't actually render the docs locally due to build issues with the Javadocs (`mvn javadoc:javadoc`) but in general Markdown supports raw HTML so these comments should work. If not I can just remove this section since it doesn't seem to be correct anyway.

Reply:
yes this does work.

@ZhuTopher the README for docs says you should run `mvn javadoc:javadoc` to generate the java doc, but that's actually an independent section and not a prerequisite of running `jekyll build`. actually i run `jekyll serve` that would convert the markdown to html and start a local server to view them.